app/models/project.rb

```diff
@@ -15,12 +15,18 @@ class Project < ApplicationRecord
 
   has_many :issues
 
+  def self.get_total_team_projects
```

Comment:
For next time, remember that rails convention for getters is that we don't append get_ with method name. Just write it as total_team_projects.